includes/frontend/class-wcmp-checkout.php

```diff
@@ -245,6 +245,26 @@ public function getDeliveryOptionsConfig(): array
                     $value += $chosenShippingMethodPrice;
                 }
 
+                if(in_array($settingName, [WCMYPA_Settings::SETTING_CARRIER_DELIVERY_MORNING_ENABLED, WCMYPA_Settings::SETTING_CARRIER_DELIVERY_EVENING_ENABLED], true)) {
+                    $ageCheckFromSettings = (bool) WCMYPA()->setting_collection->getByName(sprintf('%s_%s', $carrier, WCMYPA_Settings::SETTING_CARRIER_DEFAULT_EXPORT_AGE_CHECK));
+                    $ageCheckFromProduct = false;
+                    foreach (WC()->cart->get_cart() as $cartItem) {
+                        /**
+                         * @var WC_Product $product
+                         */
+                        $product = $cartItem['data'];
+                        $ageCheckMeta = $product->get_meta(WCMYPA_Admin::META_AGE_CHECK, true);
+                        if($ageCheckMeta === 'yes') {
+                            $ageCheckFromProduct = true;
+                            break;
+                        }
+                    }
+
+                    if($ageCheckFromSettings || $ageCheckFromProduct) {
+                        $value = 0;
```

Comment:
waarom 0 en niet false?

Reply:
Toen ik checkte kwamen settings door als 1 of 0. Daarom de 0. Maar heb het aangepast naar false want dat werkt ook en is netter.